godtools/Views/BaseTractView.swift

```diff
@@ -14,19 +14,34 @@ class BaseTractView: UIView {
     
     var data: XMLIndexer?
     var colors: TractColors?
-    var configurations = TractConfigurations()
+    var configurations: TractConfigurations?
```

Comment:
All of these, except contentView could be defined as "let" statements with no optionals. They are defined in the init() function and are not changed elsewhere in the project
````swift
let data: XMLIndexer
let colors: TractColors
let configurations: TractConfigurations
var contentView: TractRoot?
````

This is nice for two reasons:
1. It's clear that the values are not reassigned anywhere
2. We don't have to unwrap optionals everywhere we use those properties

contentView is still a var & optional because we need to access something on `self` which requires `super.init` to be called... and let's have to be initialized before `super.init` is called.